.github/workflows/build.yml

```diff
@@ -49,6 +49,15 @@ jobs:
             skip-crashpad: false
             build-appimage: true
             build-deb: true
+          - os: ubuntu-24.04
+            container: ghcr.io/chatterino/chatterino2-build-ubuntu-24.04:latest
+            qt-version: 6.6.1
```

Comment:
(not for this PR and probably for Chatterino/docker): We could bump this to 6.7.2 (as well as the Windows version).

Reply:
Actually I also thought that, but wasn't sure if there was a specific reason for it. I think it would need to be adjusted in both Chatterino/docker and Chatterino/chatterino2 (See https://github.com/Chatterino/docker/blob/master/Dockerfile_chatterino2-build-ubuntu-24.04#L97)

Reply:
Can definitely be done, just needs testing on the various distros